misc/ci.py

```diff
@@ -3,11 +3,15 @@
 import os
 
 ignores = [
+	'ninja_deps',
 	'.git/',
 	'misc/afl-fuzz-tokens/',
-	'ninja_deps',
 	'src/depfile_parser.cc',
 	'src/lexer.cc',
+	'.git\\',
```

Comment:
I would have preferred to have the python code here dynamically translate the path separators... but I do not know python very well, so doing it "properly" was beyond the level of effort that I wanted to put into this.